test/base/test_collector.py

```diff
@@ -42,30 +42,30 @@ def __init__(self, writer):
 
     def preprocess_fn(self, **kwargs):
         # modify info before adding into the buffer, and recorded into tfb
-        # if info is not provided from env, it will be a ``Batch()``.
-        if not kwargs.get('info', Batch()).is_empty():
+        # if only obs exist -> reset
+        # if obs/act/rew/done/... exist -> normal step
+        if 'rew' in kwargs.keys():
```

Comment:
```suggestion
        if 'rew' in kwargs:
```

``kwargs`` is a dict. Just use an ``in`` test.